src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -213,10 +284,21 @@ public function isVariadic(): bool
     /** Checks if the function/method contains `throw` expressions. */
     public function couldThrow(): bool
     {
+        return $this->couldThrow;
+    }
+
+    private function computeCouldThrow(MethodNode|Node\Stmt\Function_|Node\Expr\Closure|Node\Expr\ArrowFunction $node): bool
+    {
+        $statements = $node->getStmts();
+
+        if ($statements === null) {
+            return false;
```

Comment:
A test with an empty method may be missing? 🤔

Reply:
Yes, added.